LICENSE.md

```diff
@@ -87,3 +87,4 @@ Julia bundles the following external programs and libraries on some platforms:
 - [GIT](http://git-scm.com/about/free-and-open-source)
 - [ZLIB](http://zlib.net/zlib_license.html)
 - [LIBEXPAT](http://expat.cvs.sourceforge.net/viewvc/expat/expat/README)
+- [CYGWIN](https://cygwin.com/licensing.html)
```

Comment:
let's also link to https://github.com/mintty/mintty/blob/master/LICENSE since it's somewhat separate